README.md

```diff
@@ -117,6 +117,134 @@ class Test {
 ```
 
 
-## Future work
-  - Converters for `java.util.stream`
-  - [`Spliterator`](https://docs.oracle.com/javase/8/docs/api/java/util/Spliterator.html)s for Scala collections
+## Converters from Scala collections to Java 8 Streams
+
+Scala collections gain `seqStream` and `parStream` as extension methods that produce a Java 8 Stream
```

Comment:
I wonder the same thing, but I don't think it matters much to this API.  We can have the names be as above or `sequentialStream` and `parallelStream` or whatever regardless.